core/metadata-3.3.md

```diff
@@ -0,0 +1,56 @@
+---
+title: IRCv3.3 Metadata
+layout: spec
+copyrights:
+  -
+    name: "Kiyoshi Aman"
+    period: "2016"
+    email: "kiyoshi.aman@gmail.com"
+---
+## Introduction
+
+This specification incorporates the IRCv3.2 metadata specification and
+incorporates it by reference.
+
+## METADATA
+
+### METADATA PRIORITY
+
+This subcommand provides a mechanism for clients to inform the server what
+priorities to assign to particular keys. The default priority is 'on
+subscription' for all keys. The format of `METADATA PRIORITY` MUST be as
+follows:
+
+    METADATA <Scope> PRIORITY <Priority> :<Keys>
+
+`<Scope>` is one of `global`, `channel`, or `user`, reflecting all subscriptions,
+channel-related subscriptions, and subscriptions via `MONITOR`.
+
+`<Priority>` operates in descending order, with `sub` enabling on-subscription
+and on-change notifications, `change` permitting only on-change notifications,
+and `request` requiring that the client explicitly request metadata with the
+relevant keys.
+
+`<Keys>` may optionally be an asterisk (`*`), in which case the priority change
+applies to all keys.
+
+If any key is invalid, that key MUST generate an `ERR_KEYINVALID` event. If any
+key is not intended to be visible to the user due to permissions, that key MUST
+generate an `ERR_KEYNOPERMISSION` event. As usual, if the target is invalid,
```

Comment:
This should be "generate an `ERR_NOMATCHINGKEY` or `ERR_KEYNOPERMISSION` event", to stay consistent with 3.2's handling of keys that aren't visible.

[3.2 `GET`](http://ircv3.net/specs/core/metadata-3.2.html#metadata-get)
> Servers MAY replace certain metadata, which is considered not visible for the requesting user, with `ERR_NOMATCHINGKEY` or with `ERR_KEYNOPERMISSION`.